config/database.yml

```diff
@@ -0,0 +1,18 @@
+default: &default
```

Comment:
- [ ] Hei @iammouaz, unfortunately, I cannot find your **db** folder therefore I can't run your migrations. Please take a step back and see if you missed it.